tchannel/tornado/connection.py

```diff
@@ -267,11 +267,8 @@ def _handle_error_message(message):
                 future.set_exception(error)
             else:
                 error = self.response_message_factory.build(message)
-                if error:
-                    self.tchannel.event_emitter.fire(
-                        EventType.after_receive_error,
-                        error,
-                    )
+                log.error('Received error frame for %d too late %s',
```

Comment:
no "if error"?

Shouldn't cause any issues, only effect will be logging a bunch of empty errors.